cmd/snap/cmd_snap_op.go

```diff
@@ -915,7 +915,9 @@ func (x *cmdRefresh) holdRefreshes() (err error) {
 	} else {
 		dur, err := time.ParseDuration(x.Hold)
 		if err != nil {
-			return fmt.Errorf("hold value must be a number of hours or minutes (e.g., 72h): %v", err)
+			return fmt.Errorf("hold value must be a number of hours, minutes or seconds, or \"forever\": %v", err)
+		} else if dur < time.Second {
```

Comment:
Should this check actually happen at the API level or is it mostly to sugar on the CLI?

Reply:
should be in the API only then

Reply:
I'd still have this check because the API only takes time instants (not durations) and that will make the message less readable. The CLI would return `cannot hold refreshes for less than a second: -5s` instead of the API's `cannot hold refreshes because holding time has passed: 2022-11-29T16:03:00Z00:00`

Reply:
Validating times (instead of durations) is also a bit problematic because it might expire just after we validate it but before the operation returns so we still place a hold that is already expired. It might be better to validate it as a duration (because it's well-defined and might prevent trivial mistakes) and then just place whatever hold the API takes even if it's expired.

Reply:
let's leave the API alone then if it's taking  a point in time